packages/react/src/SelectPanel/SelectPanel.module.css

```
@@ -31,3 +36,45 @@
   height: inherit;
   max-height: inherit;
 }
+
+.Message {
+  display: flex;
+  height: 100%;
+  min-height: min(calc(var(--max-height) - 150px), 324px); /* maxHeight of dialog - (header & footer) */
+  padding-top: var(--base-size-24);
+  padding-bottom: var(--base-size-24);
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex-grow: 1;
+  gap: var(--base-size-4);
+
+  a {
+    color: inherit;
+    text-decoration: underline;
+  }
+}
+
+.MessageTitle {
+  font-size: var(--text-body-size-medium);
+  font-weight: var(--base-text-weight-semibold);
+}
+
+.MessageBody {
+  display: flex;
+  font-size: var(--text-body-size-medium);
```

```suggestion
  font-size: var(--text-body-size-small);
```

IIRC this should be 12px/.75rem